vnext/local-cli/generator-windows/templates/cpp/proj/MyApp.vcxproj

```diff
@@ -100,7 +93,9 @@
     <Import Project="PropertySheet.props" />
   </ImportGroup>
   <ImportGroup Label="ReactNativeWindowsPropertySheets">
-    <Import Project="$(ReactNativeWindowsDir)\PropertySheets\External\Microsoft.ReactNative.Uwp.CppApp.props" Condition="Exists('$(ReactNativeWindowsDir)\PropertySheets\External\Microsoft.ReactNative.Uwp.CppApp.props')" />
+    <Import Project="$(ReactNativeWindowsDir)\PropertySheets\external\Microsoft.ReactNative.Uwp.CppApp.props" />
+    <Import Project="$(ReactNativeWindowsDir)\PropertySheets\WinUI.props" />
```

Comment:
Could we add these to the existing external props? The motivation there was to avoid breaking template changes. This seems important if we're adding to the template for non-winui projects.

If we do make template changes that affect non-winui projects we should probably apply those to the other projects we have in the repo.